Added clarification of text for Language translations

Ambertvu opened the pull request:
## Description

I've clarified the introduction of the language file page. I noticed that some had a tough time finding the correct syntax and explanation on how to get and create the files. I hope that by first giving a short explanation and clearly describing the `key` and `area` elements it will be easier to pickup.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [x] Other

alina-tincas reviewed (changes requested):
Hi @Ambertvu, thank you for the PR 🙌 

An example with clarification is indeed nice to have! I will just run this over by our developers to confirm the example, and meanwhile I have made a suggestions to replace the community link with https://community.umbraco.com/. Let me know what you think about the suggestion 😊

Ambertvu commented:
Done and done, thanks!

alina-tincas commented on 10/umbraco-cms/extending/language-files.md:
```diff
@@ -9,11 +9,31 @@ description: >-
+Language files are XML files used to translate:
 - The Umbraco backoffice user interface so that end users can use Umbraco in their native language. This is particularly important for content editors who do not speak English.
 - The member identity errors in an Umbraco website enabling end users to use Umbraco in the website language.
+- Add translations for your packages [see here for docs on how to include translations for your own package](packages/language-files-for-packages.md)
```
To make the link text shorter and hence readable that it is a link:
```suggestion
-  Read [Add translations for your packages](packages/language-files-for-packages.md) article to see how to include translations for your own package.
```

alina-tincas commented on 11/umbraco-cms/extending/language-files.md:
```diff
@@ -6,11 +6,31 @@ description: >-
+Language files are XML files used to translate:
 - The Umbraco backoffice user interface so that end users can use Umbraco in their native language. This is particularly important for content editors who do not speak English.
 - The member identity errors in an Umbraco website enabling end users to use Umbraco in the website language.
+- Add translations for your packages [see here for docs on how to include translations for your own package](packages/language-files-for-packages.md)
```
```suggestion
-  Read [Add translations for your packages](packages/language-files-for-packages.md) article to see how to include translations for your own package.
```

alina-tincas commented on 12/umbraco-cms/extending/language-files.md:
```diff
@@ -6,11 +6,31 @@ description: >-
+Language files are XML files used to translate:
 - The Umbraco backoffice user interface so that end users can use Umbraco in their native language. This is particularly important for content editors who do not speak English.
 - The member identity errors in an Umbraco website enabling end users to use Umbraco in the website language.
+- Add translations for your packages [see here for docs on how to include translations for your own package](packages/language-files-for-packages.md)
```
```suggestion
-  Read [Add translations for your packages](packages/language-files-for-packages.md) article to see how to include translations for your own package.
```

alina-tincas reviewed (changes requested):
Hi @Ambertvu thank you for your patience! The code examples look good.

I have just one suggestion, can you take a look at it and let me know what you think? 😊

Ambertvu commented:
@alina-tincas I did a small change to your suggestion (not sure how to do it on github, so I just approved and changed them)

I removed the 'article' word.

I think it's reads better with:
'Read the xxxx article to see how..`
or this felt even better:
'Read xxxx to see how..`

and not
`Read xxx article to see how..`

alina-tincas commented:
That is all right @Ambertvu 😊 

Merging it in, thank you again for the PR 🙌

alina-tincas commented:
(as for making changes, you can do them locally on the same branch you have pushed those changes to, or you can just commit the suggestions directly in github)

Ambertvu commented:
Anytime!

Yea that's what I did, hah, thought maybe there was an other way. Thanks!
